### ISSUE TITLE
About commit【[FB] extensions | extensions can run on about: site 】 

### ISSUE THREAD

**Issue by benzBrake** [Floorp-Projects/Floorp#109]
https://github.com/Floorp-Projects/Floorp/commit/94d4f06f1aa328952266089de493aac6b94733b4

I have try this on firefox official release, only make extension work on  about:home / about:newtab / about:certerror but not about:config / about:preferences

And I think it's very necessary add 'moz-extension' to make extensions work on other extension page.

**Comment by surapunoyousei** [comment]
yes. what you are saying is correct. But it's dangerous to make it work on all about: sites.

 About:config and about:preferences have settings, and add-ons can change the settings as much as you want. So it's not allowed on purpose.

 moz-extesion was briefly introduced around 8.7.2 due to compatibility issues with Dark Reader and other add-ons, but has a history of being rejected. I have to think a little.

**Comment by surapunoyousei** [comment]
Gesturefy also has issues.  Since mouse gestures are enabled on Gesturefy's settings site, tabs are closed when mouse gestures are set, which greatly hinders the user experience.

**Comment by benzBrake** [comment]
Gesturefy is what i need, I used to use Mousegesture.uc.js to implement gestures，all page works. As I switch to gesturefy, it's a little bit inconvenience.

**Comment by surapunoyousei** [comment]
It's impossible for me to make all add-ons compatible with my technical ability. I feel a little difficult

**Comment by benzBrake** [comment]
Oh, I see, It's just an advice, you can move this issue to Discussions
